### PR TITLE
[DataBaseImporter]: do not use same parameters to generate volume id of the medData object depending on the file extension. Dicom need a different approach.

### PR DESCRIPTION
**Pull request by paulineMig**
fix https://github.com/medInria/medInria-public/issues/1008
As explained in the issue as for now medinria is not able to read Dicom correctly.
Propose fix: keep the same key id for all extension except dicom one. in this case replace serieDescription by serieNumber + echoTime + flipAngle

### CODE REVIEW

**Review by juliencastelneau — APPROVED** [review]
Ok for me
I introduced the bug few months ago when I modified the method `populateMissingMetadata`

File: src/layers/legacy/medCoreLegacy/database/medAbstractDatabaseImporter.cpp [hunk 1]
```diff
@@ -1000,13 +1005,39 @@ QString medAbstractDatabaseImporter::generateUniqueVolumeId ( const medAbstractD
...
     return key;
 }
+
+//-----------------------------------------------------------------------------------------------------------
+/**
+* Look for dicom extension in file name
+* @param fileName - name of the file that we try to import
+* @return  true if dicom file, false otherwise
+**/
+bool medAbstractDatabaseImporter::isDicomName(const QString & fileName)
```
Comment (Florent2305) [inline src/layers/legacy/medCoreLegacy/database/medAbstractDatabaseImporter.cpp:1035]: Please replace by
```C++
bool medAbstractDatabaseImporter::isDicomName(const QString & fileName)
{
    QFileInfo info(fileName);
    return info.suffix().toLower() == "dcm";
}

**Review by Florent2305 — CHANGES_REQUESTED** [review]
Thank you for this contribution.
Can you take into account my small change.

File: src/layers/legacy/medCoreLegacy/database/medAbstractDatabaseImporter.cpp (hunk 1 above)
Comment (paulineMig) [reply src/layers/legacy/medCoreLegacy/database/medAbstractDatabaseImporter.cpp:1035]: Done and good catch I do not know why i didn't do it like that from the start

Comment (Florent2305) [reply src/layers/legacy/medCoreLegacy/database/medAbstractDatabaseImporter.cpp:1035]: Thanks

**Review by Florent2305 — APPROVED** [review]
ok

**Pull request merged**